backend/ipp.c

```diff
@@ -1914,7 +1911,7 @@ main(int  argc,				/* I - Number of command-line args */
 
     if (ipp_status <= IPP_STATUS_OK_CONFLICTING && argc > 6)
     {
-      fprintf(stderr, "PAGE: 1 %d\n", copies_sup ? atoi(argv[4]) : 1);
+      fprintf(stderr, "PAGE: 1 %ld\n", copies_sup ? strtol(argv[4], NULL, 10) : 1);
```

Comment:
As "copies" is limited to a 32-bit integer (IPP syntax), and more specifically to a range of 1 to 999 with the default configuration, I'd rather stick with atoi.